.buildkite/commands/installable-build-wordpress.sh

```diff
@@ -14,4 +14,4 @@ echo "--- :closed_lock_with_key: Installing Secrets"
 bundle exec fastlane run configure_apply
 
 echo "--- :hammer_and_wrench: Building"
-bundle exec fastlane build_and_upload_installable_build
+bundle exec fastlane build_and_upload_app_store_connect
```

Comment:
I thought we'd want installable builds to go out via App Center? 

In case this is a typo:

```suggestion
bundle exec fastlane build_and_upload_app_center skip_confirm:true
```

But then, I wonder, what would be the difference between this command and the new `.buildkite/commands/release-build-wordpress-internal.sh`?

Reply:
Shoot – I rebased this PR and I think I messed up the merge pretty good. Sorry for the confusion – I've fixed it in edcfabc45bb3afbaa468a767c57af2c6be33d2a8.